site/roles/trinity/zabbix_repo/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+zabbix_repo_repos:
```

Comment:
The whole `zabbix_repo` role has somehow sneaked in, while it should have been handled via dependencies as it is done in `zabbix/meta/main.yaml`.